README.md

```diff
@@ -62,6 +62,25 @@ Optionally, you can add a similar example (i.e. without the mcp key) to a file c
 }
 ```
 
+If you are running vscode server remotely (e.g. WSL2 on Windows) then use the Remote Settings (JSON) file instead. You can do this by pressing `Ctrl + Shift + P` and typing `Preferences: Open Remote Settings (JSON)`. Default location is `~/.vscode-server/data/Machine/settings.json`.
```

Comment:
Hi @richeney, thanks for the PR! recommend adding this one line after the usage with VScode section without the json block below (because it'll be the same)

Reply:
Makes sense, although the remote settings was empty by default for me so figured it might be easier. I really fumbled through this, although I am now just using .vscode/mcp.json now that it is supporting remotes better,